datacamp_python_project_spec_lefthandedness.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Do left-handed people really die young?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Project info"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Project title**: Do left-handed people really die young?\n",
+    "\n",
+    "**Name:** Madeleine Bonsma-Fisher\n",
+    "\n",
+    "**E-mail:** madeleine.bonsma@gmail.com\n",
+    "\n",
+    "**GitHub username**: mbonsma\n",
+    "\n",
+    "**Link to prior writing**: \n",
```

Comment:
Ahmed already added this on the other notebook so not sure you need it here also, but maybe just copy that link over.